images/tcp-stream.proto

```diff
@@ -16,9 +21,6 @@ message tcp_stream_entry {
 	optional uint32		rcv_wscale	= 8;
 	optional uint32		timestamp	= 9;
 
-	optional bool		cork		= 10;
```

Comment:
Doesn't it break backward compatiblity? Have you tried to restore from old images?

Reply:
Yes, this break backwards compatibility.

But if we don't move the TCP socket options out of the TcpStreamEntry, it is difficult to restore the TCP socket options for unconnected TCP sockets.

How about this? We continue to support restoring TCP socket options from TcpStreamEntry and SkOptsEntry, but dumping TCP socket options to TcpOptsEntry in new versions of CRIU.

Reply:
> Yes, this break backward compatibility.

We can't do that.

> How about this? We continue to support restoring TCP socket options from TcpStreamEntry and SkOptsEntry, but dumping TCP socket options to TcpOptsEntry in new versions of CRIU.

It is what we are usually doing in such cases.

Reply:
fixed